doc/contributor/howto-guide-set-up-development-environment.md

```diff
@@ -74,6 +74,9 @@ cargo fmt && cargo clippy --profile=test -- --deny warnings && cargo test
 git status # Shows any diffs created by `cargo fmt`
 ```
 
+If you are seeing errors when running locally that are not present in the CI,
+you may need to update your local rust version.
```

Comment:
Should we bump the minimum version here to one that doesn't yield errors?

https://github.com/googleapis/google-cloud-rust/blob/dd9e4da76c89d9cdf05d18271dfcc2573f847bed/doc/contributor/howto-guide-set-up-development-environment.md?plain=1#L12